src/components/date-picker/DatePicker.js

```diff
@@ -195,6 +198,7 @@ class DatePicker extends React.Component<Props> {
             setDefaultDate: true,
             defaultDate: defaultValue,
             field: this.dateInputEl,
+            firstDay: localesWhereWeekStartsOnSunday.includes(getCookie('lang') || 'en-US') ? 0 : 1,
```

Comment:
Can you add a unit test for this change?

Reply:
> Can you add a unit test for this change?

The DatePicker test is changed to `DatePicker.test.js.fixme`. So I want to hear @alexkrolick 's feedback first (author of #1572), then fix the tests.